diff --git src/core/BUILD src/core/BUILD
index f4f0431196..d7084aaed4 100644
--- src/core/BUILD
+++ src/core/BUILD
@@ -2222,10 +2222,6 @@ grpc_cc_library(
             "posix_event_engine",
             "cf_event_engine",
         ],
-        "//:mac_arm64": [
-            "posix_event_engine",
-            "cf_event_engine",
-        ],
         "//:ios": ["cf_event_engine"],
         "//:tvos": ["cf_event_engine"],
         "//:watchos": ["cf_event_engine"],
