diff --git bazel/repositories.bzl bazel/repositories.bzl
index 6d619e6..0195dc8 100644
--- bazel/repositories.bzl
+++ bazel/repositories.bzl
@@ -70,5 +70,8 @@ def prometheus_cpp_repositories():
             "https://storage.googleapis.com/bazel-mirror/zlib.net/zlib-1.2.13.tar.gz",
         ],
         build_file = "@com_github_jupp0r_prometheus_cpp//bazel:zlib.BUILD",
+        patches = [
+            "@com_github_ray_project_ray//thirdparty/patches:zlib-fdopen.patch",
+        ]
     )

